elements/pl-dropdown/pl-dropdown.mustache

```diff
@@ -12,7 +12,7 @@
     <div id="pl-dropdown-{{uuid}}" class="d-inline-block form-group form-inline">
         <select name="{{answers-name}}" class="form-control">
             {{#options}}
-                <option value="{{value}}" {{#selected}}selected{{/selected}}>{{value}}</option>
+                <option value="{{value}}" {{#selected}}selected{{/selected}}>{{{value}}}</option>
```

Comment:
Does the value attribute need triple braces also?

Reply:
That's not even enough. There are some problems when the value submitted in the form includes the &. Reviewing now.

Reply:
It works properly without unescaping value. I added the change in submitted and correct panels as well.